storage/notifications.go

```diff
@@ -91,6 +92,30 @@ func toNotification(rn *raw.Notification) *Notification {
 	return n
 }
 
+func toNotificationFromProto(pbn *storagepb.Notification) *Notification {
+	n := &Notification{
+		ID:               pbn.Name,
```

Comment:
It seems like elsewhere we have used getter functions for this (e.g. pbn.GetName()) rather than accessing attrs directly. @noahdietz do you have a preference? It'd be good to be consistent probably.

Reply:
Yes! Thanks for catching this. We should use the protobuf accessors everywhere rather than direct access. Please update all such instances.

Reply:
good catch, updated